src/App/Fossa/Init/.fossa.yml

```diff
@@ -0,0 +1,69 @@
+# # fossa-cli configuration file.
```

Comment:
Double commented, so users can `Ctrl+A` and uncomment in their code editors, while still retaining helpful links!